modules/PhyVersoBSP/rcd_1/ac_rcdImpl.cpp

```diff
@@ -7,17 +7,23 @@ namespace module {
 namespace rcd_1 {
 
 void ac_rcdImpl::init() {
+    mod->serial.signal_error_flags.connect([this](int connector, ErrorFlags error_flags) {
+        if ((connector == 1) && error_flags.rcd_triggered) {
+            EVLOG_info << "[1] RCD triggered: " << error_flags.rcd_triggered;
+            // TODO: publish somewhere
```

Comment:
This could probably raise a ac_rcd/AC error: https://github.com/EVerest/everest-core/blob/main/errors/ac_rcd.yaml

Reply:
I have changed the code to raise a ac_rcd/DC error because the phyVERSO is specified to use a residual direct current monitoring device, so normally this would be blind to AC fault currents. I could also change it to MREC2GroundFailure error because this error seems to not discriminate between AC or DC faults in case one does not only want to attach, like specified, a RDC-MD but some kind of monitoring device that is not blind to AC fault currents.